js/brushes/toad.js

```diff
@@ -0,0 +1,9 @@
+/* global mouseIsPressed, image, mouseX, mouseY */
+window.brushes.toad = {
+  name: 'toad',
+  draw: function () {
+    if (mouseIsPressed) {
+      image('toad', mouseX, mouseY, 50, 50);
```

Comment:
the first argument to `image()` should be the name of the variable (an object created using `loadImage()`) containing the image data. what u passed was the string `'toad'` (essentially the word or text) rather than the actual variable `toad` (without quotes)

Reply:
changed that and it should be working now